demo/index.html

```diff
@@ -42,14 +42,17 @@
 </head>
 <body>
 
+  <paper-toast class="fit-bottom" text="hi there!" opened></paper-toast>
+
   <paper-button raised onclick="document.querySelector('#toast1').show()">Discard Draft</paper-button>
 
   <paper-button raised onclick="document.querySelector('#toast2').show()">Get Messages</paper-button>
 
   <paper-toast id="toast1" text="Your draft has been discarded."></paper-toast>
 
-  <paper-toast id="toast2" text="Connection timed out. Showing limited messages." duration="0" onclick="this.hide()">
+  <paper-toast id="toast2" text="Connection timed out. Showing limited messages." duration="0">
     <span role="button" tabindex="0" style="color: #eeff41;margin: 10px" onclick="console.log('RETRY')">Retry</span>
+    <span role="button" tabindex="0" style="color: #eeff41;margin: 10px" onclick="document.querySelector('#toast2').hide()">Dismiss</span>
```

Comment:
ugh, this should not be in the demo page since it is not conform to the [design spec] (https://www.google.com/design/spec/components/snackbars-toasts.html#snackbars-toasts-usage)